DTS-STN/Service-Canada-Labs
# Next 13 Upgrade & Fix OAS Estimator page forcing full-reload in dev

# Description

This PR include two things:

1. A fix to the full-reload of the OAS page in dev (**export function was not capitalized**)
2. Upgrade to Next 13 with `<Link>` components migrated to the new format.

I upgraded to Next 13 during my debugging of the first issue and when it was solved differently, I figured we could keep this in for when we might move to the new Next 13 project structure. Everything seems to be working very well on Next 13 and we can adopt new features piece by piece. Here's some stuff to read that you might find interesting:

[Next13 Upgrade Guide](https://beta.nextjs.org/docs/upgrade-guide)

[App Router features (this is the new routing/project structure setup of Next)](https://beta.nextjs.org/docs/app-directory-roadmap)

As of right now nothing much has changed but eventually we should look into adopting the App router 👍 

## Acceptance Criteria

See that the site runs without errors after installing new dependencies.

## Test Instructions

1. yarn install
4. yarn dev, ensure dev server runs without breaking errors on pages or components
5. yarn build
6. yarn start, ensure dev server runs without breaking errors on pages or components

## Checklist
